Zend/zend_vm_gen.php

```diff
@@ -19,7 +19,7 @@
 	 $Id$
 */
 
-$header_text = <<< DATA
+const HEADER_TEXT = <<< DATA
```

Comment:
Also - not really sure why the version number would matter [unless we're worried about people using PHP 4](https://3v4l.org/P6eW4). :)